javascript-bank/bank.js

```diff
@@ -0,0 +1,37 @@
+/* exported Bank */
+function Bank() {
+  this.accounts = [];
+  this.nextAccountNumber = 1;
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if ((balance > 0) && (balance % 1 === 0)) {
+    let acc = new Account(balance, holder);
+    acc.nextAccountNumber = this.nextAccountNumber;
+    acc.balance = balance;
+    this.accounts.push(acc);
+    this.nextAccountNumber +=1;
+    return (this.nextAccountNumber - 1)
+  } else {
+    return null
+  }
+};
+
+Bank.prototype.getAccount = function (number) {
+  let account = null;
+  for (let acc of this.accounts) {
+    if (acc.nextAccountNumber === number) {
```

Comment:
There should not be a `nextAccountNumber` property on `Account`s.

---

![image](https://user-images.githubusercontent.com/16143771/123366308-d85a3580-d52c-11eb-9110-ba34e04f11fc.png)